src/app/api/onboarding/route.ts

```diff
@@ -1,14 +1,41 @@
 import { formSchema } from "@/shared/onboarding";
 import { NextResponse } from "next/server";
+import { prisma } from "@/shared/db";
 
 export async function POST(request: Request) {
   try {
-    const data = await request.json();
+    const dataWithId = await request.json();
+    const { data } = dataWithId;
+    const { id } = dataWithId;
     const formValues = formSchema.safeParse(data);
 
     if (!formValues.success)
       return new NextResponse(JSON.stringify(formValues), { status: 403 });
-    return new NextResponse(JSON.stringify(formValues.data));
+
+    try {
+      await prisma.user.update({
```

Comment:
This should not be here. We must create a function to access the `@/shared/db` files, and just call this function here.

If we implement this logic here, we're leaking the abstracting, exposing implementation details. 
This is also bad because we spread DB logic all over the place, rather than having a centralized placed for all DB interations

Reply:
Done. Thanks for the review, thats the way.